source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -239,7 +233,7 @@ void BioXASAppController::registerClasses()
 	AMDbObjectSupport::s()->registerClass<BioXASXASScanConfiguration>();
 }
 
-void BioXASAppController::setupExporterOptions()
+void BioXASAppController::setupBeamlineExporterOptions()
```

Comment:
Small detail. The beamline itself doesn't really export anything, what do you think of `setupApplicationExporterOptions()` or `setupDataExporterOptions()`?